package.js

```diff
@@ -99,24 +95,12 @@ async function startPack() {
 function pack(plat, arch, cb) {
   // there is no darwin ia32 electron
   if (plat === 'darwin' && arch === 'ia32') return;
+  // armv7l does not exist on any platform but linux
+  if (plat !== 'linux' && arch === 'armv7l') return;
 
-  const iconObj = {
-    icon: DEFAULT_OPTS.icon + (() => {
-      let extension = '.png';
-      if (plat === 'darwin') {
-        extension = '.icns';
-      } else if (plat === 'win32') {
-        extension = '.ico';
-      }
-      return extension;
-    })()
-  };
```

Comment:
This is done automatically inside Electron Packager, see [`normalizeExt`](https://github.com/electron-userland/electron-packager/blob/65c5b6790804e365a4278d2cc57ee5be3f5437cd/common.js#L265-L281).